server/settings.py

```diff
@@ -229,6 +229,10 @@ def env(variable, fallback_value=None):
         'thumbnail': {'width': 220, 'height': 120},
         'viewImage': {'width': 640, 'height': 640},
         'baseImage': {'width': 1400, 'height': 1400},
+
+        'preview': {'width': 1024, 'height': 768},
```

Comment:
I think those won't make it to ui, it should be part of image cropping vocabulary